Makefile

```diff
@@ -5,10 +5,11 @@ DOCKER_ORG=interconnectedcloud
 PWD=$(shell pwd)
 
  # This is the latest version of the Qpid Dispatch Router
-DISPATCH_VERSION=1.16.0
+DISPATCH_VERSION=1.16.x
```

Comment:
I'd be inclined to update to 1.16.1 once the release is out. That way it is clear what it refers to. However I have no strong objection to building from branch.

Reply:
I have updated it to `1.16.1-rc1` and added logic to skip updating the latest
tag when version contains `rc`.